src/libcore/macros.rs

```diff
@@ -519,54 +519,69 @@ macro_rules! unreachable {
     });
 }
 
-/// Indicates unfinished code.
+/// Indicates unfinished code by panicking with a message of "not yet implemented".
```

Comment:
I thought this might be good for users to know they'll get a default message so they don't need to do `unimplemented!("not yet implemented")`. I had to look at the source code to see that (before having used it). Do you think that's not useful?